FAILED TEST LOG
  ✘ You should be connected to "second_database"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(re.test(lastLog))

at Context.<anonymous> (test/80.test.js:9:5)
FAILED TEST LOG
  ✘ You should be connected to "second_database"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(re.test(lastLog))

at Context.<anonymous> (test/80.test.js:9:5)
FAILED TEST LOG
  ✘ You should display the details of "second_table"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/Table"public\.second_table"/.test(lastQueryResult))

at Context.<anonymous> (test/160.test.js:8:5)
FAILED TEST LOG
  ✘ "second_table" should not have a column named "age"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/210.test.js:24:7)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ You should use "SELECT" to view all the data in "second_table"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(selectAll.test(lastLog) && !/ERROR/.test(lastLog))

at Context.<anonymous> (test/290.test.js:9:5)
FAILED TEST LOG
  ✘ You should display the tables with suggested command

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(d.test(lastLog))

at Context.<anonymous> (test/420.test.js:9:5)
FAILED TEST LOG
  ✘ "second_database" should not have a table named "second_table"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/430.test.js:35:7)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ "second_database" should not have a table named "second_table"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/430.test.js:35:7)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ You should list the databases

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/Listofdatabases/.test(lastQueryResult))

at Context.<anonymous> (test/450.test.js:8:5)
FAILED TEST LOG
  ✘ "mario_database" should have a table named "characters"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/540.test.js:19:7)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ You should use "SELECT" to view all the data in "characters"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(selectAll.test(lastLog) && !/ERROR/.test(lastLog))

at Context.<anonymous> (test/620.test.js:9:5)
FAILED TEST LOG
  ✘ The "character_actions" table should have three rows for Bowser's actions

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/1570.test.js:20:7)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ You should view all the data from "characters" and "sounds" with a JOIN statement

TypeError: Cannot read properties of null (reading 'length')
at Context.<anonymous> (test/1670.test.js:8:46)
